packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -456,6 +459,7 @@ export class AccountHandler {
     const service = form.service || query.service;
     const preVerified = !!form.preVerified;
     const verificationMethod = form.verificationMethod;
+    const atLeast18AtReg = form.atLeast18AtReg || null;
```

Comment:
With that `||` a false is now saved as a null.  :old-man-yells-at-javascript:

Reply:
As I was working on fixing this, I started to wonder if it might actually be preferable to store this as null if `isAtLeast18AtReg` returns false. Moz Social would be re-asking for age anyway if the value was false or null, and this way we don't store a clear "under18" value - null could mean 'under18', 'older account' or 'coppa disabled'. In any of these cases, RPs should recheck age if they need to know if a user is over 18. 🤔

Reply:
Having a null to represent one or more of those things is fine by me if we truly don't need to know about the false case.